libs/pages/environments/ui/src/lib/components/deployment-rules/deployment-rule-item/deployment-rule-item.tsx

```diff
@@ -0,0 +1,125 @@
+import { Icon, Menu, MenuData, Skeleton } from '@console/shared/ui'
+import { ENVIRONMENTS_DEPLOYMENT_RULES_URL, ENVIRONMENTS_URL, upperCaseFirstLetter } from '@console/shared/utils'
+import { useState } from 'react'
+import { useParams } from 'react-router'
+
+export interface DeploymentRuleItemProps {
+  id: string
+  name: string
+  startTime: string
+  stopTime: string
+  weekDays: string[]
+  isLast: boolean
+  isLoading?: boolean
+  removeDeploymentRule: (id: string) => void
+}
+
+export function DeploymentRuleItem(props: DeploymentRuleItemProps) {
+  const { id, name, startTime, stopTime, weekDays, isLast = false, isLoading = false, removeDeploymentRule } = props
+  const [menuOpen, setMenuOpen] = useState(false)
+  const { organizationId, projectId } = useParams()
+
+  const getTime = (date: string) => {
+    const converted = new Date(date)
+    const hours = converted.getUTCHours()
+    const minutes = converted.getUTCMinutes()
+    return `${hours < 10 ? '0' + hours : hours}:${minutes < 10 ? '0' + minutes : minutes}`
+  }
+
+  const isWeekdays = (): boolean => {
+    const weekdays = ['MONDAY', 'TUESDAY', 'WEDNESDAY', 'THURSDAY', 'FRIDAY']
+
+    const checkIfWeekdays = weekdays.every((weekday) => {
+      return weekDays.includes(weekday)
+    })
+
+    return checkIfWeekdays
+  }
+
+  const removeRule = (id: string) => {
+    removeDeploymentRule(id)
+  }
+
+  const menu: MenuData = [
+    {
+      items: [
+        {
+          name: 'Edit',
+          contentLeft: <Icon name="icon-solid-pen" className="text-brand-400 text-sm" />,
+          link: {
+            url: `${ENVIRONMENTS_URL(organizationId, projectId)}${ENVIRONMENTS_DEPLOYMENT_RULES_URL}/edit/${id}`,
+          },
+        },
+      ],
+    },
+    {
+      items: [
+        {
+          name: 'Delete rule',
+          textClassName: '!text-error-500',
```

Comment:
Not use important here in the composant

In `MenuItem` you can:
- Add by default replace `textClassName = ''` by `textClassName = 'text-text-500'`